cypress/integration/e2e/components/alert-dialog.spec.ts

```diff
@@ -1,4 +1,4 @@
-describe('Alert-dialog component documentation page', () => {
+describe('Alert-dialog component documentation page', function () {
```

Comment:
❓ If we use anon functions instead of arrow functions, we can leverage Mocha's context and pull the test name directly from `this`, rather than having to hard-code test names. What do folks think?